src/imputation/tmi_imputation.py

```diff
@@ -484,12 +492,78 @@ def run_tmi(
     # TMI Step 5: Calculate headcount and employment totals
     final_df = calculate_totals(expanded_df)
 
-    # add short forms back to dataframe
-    full_df = pd.concat([final_df, shortform_df])
+    TMILogger.info("TMI long form imputation completed.")
+    return final_df, qa_df
+
+
+def run_shortform_tmi(
+    shortform_df: pd.DataFrame,
+    config: Dict[str, Any],
+) -> pd.DataFrame:
+    """Function to run imputation end to end and returns the final
+    dataframe back to the pipeline
+        dataframe back to the pipeline
+    Args:
+        shortform_df (pd.DataFrame): the dataset filtered for long form entries
+        config (Dict): the configuration settings
+    Returns:
+        pd.DataFrame: dataframe with the imputed valued added
+    """
+    TMILogger.info("Starting TMI short form imputation.")
+
+    sf_target_variables = list(config["breakdowns"])
+
+    df = tmi_pre_processing(shortform_df, sf_target_variables)
+
+    mean_dict, qa_df = create_mean_dict(df, sf_target_variables, config)
+
+    # qa_df.set_index("qa_index", drop=True, inplace=True)
+
+    qa_df = qa_df.drop("trim_check", axis=1)
+
+    shortform_tmi_df = apply_tmi(df, sf_target_variables, mean_dict, formtype_short)
+
+    TMILogger.info("TMI imputation completed.")
```

Comment:
Should be `TMILogger.info("Short form TMI imputation completed.")` or something similar (and consistent with long form)